src/context/WalletProvider/NativeWallet/components/NativeCreate.tsx

```diff
@@ -19,6 +19,13 @@ import { Text } from 'components/Text'
 
 import { NativeSetupProps } from '../types'
 
+export const getVault = async (): Promise<Vault> => {
```

Comment:
Why does this need to be exported?

Reply:
It's used in `cypress/helpers.ts`. That said, that helper it is used in is no longer needed after a recent refactor, so this has been addressed in https://github.com/shapeshift/web/pull/849/commits/857b2a63e5a58f59cd6bcc0b5f825835dc3c87c3.